consensus/src/chained_bft/chained_bft_smr_test.rs

```diff
@@ -388,20 +375,6 @@ fn basic_commit_and_restart() {
                 }
             }
         }
-        // Because of the race, we can't assert the commit reliably, instead we assert
-        // both nodes commit to at least round 17.
-        // We cannot reliable wait for the event of "commit & prune": the only thing that we know is
-        // that after receiving the vote for round 20, the root should be at least height 16.
-        assert!(
-            nodes[0].smr.block_store().unwrap().root().height() >= 16,
```

Comment:
we lose some assertion here, we probably can test the round instead

Reply:
yup, for sure.